test/blackbox-tests/test-cases/action-modifying-a-dependency/dune

```diff
@@ -0,0 +1,4 @@
+(alias
+ (name   x)
+ (deps   data (universe))
```

Comment:
This also writes to `data`, so should `data` be a target in this test?

Reply:
That would create a circular dependency